new-lamassu-admin/src/components/Tooltip.js

```diff
@@ -51,4 +51,35 @@ const Tooltip = memo(({ children, width, Icon = HelpIcon }) => {
   )
 })
 
-export default Tooltip
+const HoverableTooltip = memo(({ parentElements, children, width }) => {
```

Comment:
There's a good chunk of repeated code within Tooltip and HoverableTooltip. Creating another component used by both of them with the common parts or parameterizing the behaviour could clean it up.

Reply:
I agree, My solution was creating a hook which can be used by other Tooltip-like components, regarding the popper handling